lab_3_ann_retriever/tests/advanced_search_engine_test.py

```diff
@@ -1,6 +1,6 @@
 # pylint: disable=duplicate-code, protected-access
 """
-Checks the third lab's AdvancedSearchEngine class.
+Checks the third lab's Priority КD Tree Retriever class.
```

Comment:
why do you have these changes here?

Reply:
I think it was before some editing of tests because of errors